src/main/zabbix/etc/zabbix/scripts/sonicmq_env.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Environment settings for running SonicMQ Zabbix scripts
+
+# Location of SonicMQ client libraries
+SONICMQ_JAR_PATH=/opt/zabbix-sonicmq-monitoring/lib
+
+# Installation path of monitor application
+SMQ_MON_INSTALL_PATH=/opt/zabbix-sonicmq-monitoring
+
+# Monitor output file
+SMQ_MON_OUTPUT_FILE=/tmp/sonicmq-monitor-output.json
+
+# Monitor configuration file, configures SonicMQ connection and monitor application
+SMQ_MON_CONFIG_FILE=$SMQ_MON_INSTALL_PATH/config.json
+
+# Maximum monitor output file age in seconds
+SMQ_MON_OUTPUT_MAX_AGE=120
+
+# Run mode: in cron mode scripts do not trigger updates and cron job is expected to be setup for it
+SMQ_MON_CRON_MODE=0
+
+# Monitor application log
+# Define either absolute path or special value "syslog" to redirect to syslog
+SMQ_MON_LOG=syslog
+
+echo $SMQ_MON_LOG
```

Comment:
I bet this does not work well together with zabbix scripts? Should be commented out?

Reply:
Debug echo accidentally left here -> will remove.